packages/prover/src/utils/evm.ts

```diff
@@ -67,35 +72,55 @@ export async function getVMWithState({
     storageKeysMap[address] = storageKeys;
   }
 
-  if (storageKeysMap[tx.from ?? ZERO_ADDRESS] === undefined) {
-    storageKeysMap[tx.from ?? ZERO_ADDRESS] = [];
+  // If from address is not present then we have to fetch it with for all keys
```

Comment:
either

```suggestion
  // If from address is not present then we have to fetch it for all keys
```

or 

```suggestion
  // If from address is not present then we have to fetch it with all keys
```

there is another similar case below